src/test/ui/panic-runtime/abort-link-to-unwind-msvc-no-std-link-to-libcmt.rs

```diff
@@ -0,0 +1,25 @@
+// run-fail
+// compile-flags: -C panic=abort -C target-feature=+crt-static
```

Comment:
I'm slightly confused about `-C target-feature=+crt-static`. We're statically linking the crt but also dynamically linking `libcmt`. Shouldn't we be testing on or the other.

Reply:
`libcmt` is the static C runtime library. It is a static library, though it should be linked dynamically. I just cannot tell why it needs dynamic linking. It should make no difference between dynamic linking and static linking.

Reply:
`+crt-static` will [already link `libcmt`](https://github.com/rust-lang/libc/blob/708e5284c1586b2e5f5fea876af96fc484334055/src/windows/mod.rs#L241) so there should be no reason to declare it manually.

Reply:
That's right when using `libc` crate, but here we need to test under `no_std`.